Classes/ViewHelpers/Form/VariantSelectViewHelper.php

```diff
@@ -166,6 +168,18 @@ protected function getOptions(): array
             $disabled = '';
             if (!$beVariant->getIsAvailable() && $beVariant->getProduct()->isHandleStockInVariants()) {
                 $disabled = 'disabled';
+
+                if ($this->arguments['showOutOfStockMessage']) {
+                    $labelOutofStock = LocalizationUtility::translate(
```

Comment:
We do not need this variable, but OK for now.

Reply:
You decide. I like readability but you're the maintainer :)